src/hotspot/share/gc/shenandoah/shenandoahHeap.cpp

```diff
@@ -1792,14 +1792,16 @@ void ShenandoahHeap::stw_unload_classes(bool full_gc) {
   if (!unload_classes()) return;
   // Unload classes and purge SystemDictionary.
   {
-    ShenandoahGCPhase phase(full_gc ?
-                            ShenandoahPhaseTimings::full_gc_purge_class_unload :
-                            ShenandoahPhaseTimings::degen_gc_purge_class_unload);
+    ShenandoahPhaseTimings::Phase p = full_gc ?
```

Comment:
Please name the local variable `phase`. `p` is usually a "location" around GC code.